Backends/Wayland/package.lisp

```diff
@@ -0,0 +1,39 @@
+(in-package #:common-lisp-user)
+
+(uiop:define-package #:clim-wayland
+    (:use #:clim #:clim-lisp #:clim-backend)
+  (:local-nicknames (#:alx #:alexandria)
+                    (#:wlc #:com.andrewsoutar.cl-wayland-client)
+                    (#:wl-core #:com.andrewsoutar.cl-wayland-client/core)
+                    (#:wl-egl #:com.andrewsoutar.cl-wayland-client/egl-core)
+                    (#:xdg #:com.andrewsoutar.cl-wayland-client.protocol/stable/xdg-shell/xdg-shell))
+  (:import-from #:climi
+                #:standard-port
```

Comment:
head-ups - standard-port functionality will be merged with basic-port, so this class won't exist soon (see the branch modules)

Reply:
I think this is done upstream now and I've since rebased so that it's included.